components/filter/index.js

```diff
@@ -20,7 +22,11 @@ const Filter = (props) => {
     >
       <div className="icon">
         <Image
-          src="/assets/filter-icon.svg"
+          src={
+            theme === 'light'
+              ? '/assets/filter-icon.svg'
+              : '/assets/filter-icon-dark.svg'
+          }
           alt="Filter icon"
           width={20}
```

Comment:
width and height should come from constant. As well, the UL should come dynamic. create an Object with the list of options required... coop over it to create a List and then bind the event on it. Avoid doing any hardcoding of the content